webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -401,235 +311,175 @@ const AdvanceTextEditor = ({
         } else {
             setShowFormattingSpacer(false);
         }
-    }, [message, input]);
-
-    const handleKeyDown = (e: React.KeyboardEvent<TextboxElement>) => {
-        const ctrlOrMetaKeyPressed = e.ctrlKey || e.metaKey;
-        const ctrlEnterKeyCombo = (ctrlSend || codeBlockOnCtrlEnter) &&
-            Keyboard.isKeyPressed(e, KeyCodes.ENTER) &&
-            ctrlOrMetaKeyPressed;
-
-        const ctrlKeyCombo = Keyboard.cmdOrCtrlPressed(e) && !e.altKey && !e.shiftKey;
-        const ctrlAltCombo = Keyboard.cmdOrCtrlPressed(e, true) && e.altKey;
-        const shiftAltCombo = !Keyboard.cmdOrCtrlPressed(e) && e.shiftKey && e.altKey;
-        const ctrlShiftCombo = Keyboard.cmdOrCtrlPressed(e, true) && e.shiftKey;
-
-        // fix for FF not capturing the paste without formatting event when using ctrl|cmd + shift + v
-        if (e.key === KeyCodes.V[0] && ctrlOrMetaKeyPressed) {
-            if (e.shiftKey) {
-                isNonFormattedPaste.current = true;
-                timeoutId.current = window.setTimeout(() => {
-                    isNonFormattedPaste.current = false;
-                }, 250);
-            }
-        }
+    }, [Boolean(draft.message)]);
```

Comment:
Why change this? It seems like this should be recalculating when the width of the draft message changes

Reply:
This is just the creation of the callback. We don't need to create a new callback every time the message changes. Whether this is executed or not is handled somewhere else, not here.